build-scripts/build-custom-app.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+# Define the content for the main.tsx file
+read -r -d '' MAIN_CONTENT <<EOM
+
+import React, { FC, ReactNode } from "react";
+import Header from "@components/headers/Main";
```

Comment:
I can see the name of this file `src/components/headers/Main.tsx` is 'Main.tsx' - I wonder if we want to rename that file and all its references to 'Header' for consistency? /cc @PatFawbertMills

Reply:
I think we can use an index.ts file and just export it that way so the import would be `import {Header} from "@components/headers"` but that would need to be consistent across the board

Reply:
We'll leave it for now - but I'll raise a ticket to review this at some point :)

Reply:
https://linear.app/climate-policy-radar/issue/PDCT-1388/determine-file-naming-convention-and-update-code-to-align